lib/viber/platform.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const setup = require('./setup');
+const reply = require('./reply');
+const parse = require('./parse');
+const template = require('./format-message');
+
+class ViberPlatform {
+  constructor(options) {
+    this.options = options || {};
+
+    this.export = {
+      viberReply: this.reply,
+      viberTemplate: this.template
+    };
+  }
+
+  setup(api, bot, logError, optionalParser, optionalResponder) {
+    const setupFunction = setup.bind(this);
+    return setupFunction(api, bot, logError, optionalParser, optionalResponder);
+  }
+
+  reply(receiver, authToken, messageObj) {
+    return reply(receiver, authToken, messageObj);
+  }
+
+  parse(messageObject) {
+    const parseFunction = parse.bind(this);
+    return parseFunction(messageObject);
+  }
+
+  template() {
+    return template;
+  }
+}
+
+module.exports = ViberPlatform;
```

Comment:
if we compare the `platform.js` for all of these platforms we come to an identical structure, with the difference in just its parameters and enabled `template()`
~
maybe it can be a single object, with different params and a templates enabled or disabled